docs/developer-docs/daos/nns/proposal-voting.mdx

```diff
@@ -0,0 +1,174 @@
+---
+keywords: [intermediate, governance, concept, nns]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Voting on proposals
+
+<MarkdownChipRow labels={["Intermediate", "Governance", "Concept"]} />
+
+The NNS DAO is an open governance system where everyone can become a participant by staking some ICP in a neuron. Thus, any user can get voting rights by [staking some ICP in a neuron](https://internetcomputer.org/docs/current/developer-docs/daos/nns/nns-app-quickstart#stake-icp-utility-tokens-in-a-neuron) and locking the tokens for at least 6 months (called the neuron's dissolve delay).
+
+## Finding proposals and discussions
+
+You can see all the NNS proposals on the [Internet Computer dashboard](https://dashboard.internetcomputer.org/governance) or on the [NNS dapp](https://nns.ic0.app/). Voters can freely discuss proposal anywhere they like. A lot of NNS proposals are discussed on the [developer forum](https://forum.dfinity.org/c/roadmap/29).
+
+## Direct voting
+When a user has an eligible neuron and would like to vote on an open proposal, one option is to directly vote to adopt or reject the proposal. Currently, the most user-friendly way to first find the open proposals that a user can vote on and to then cast a vote is via the NNS Frontend dapp: https://nns.ic0.app/ as explained [here](https://internetcomputer.org/docs/current/developer-docs/daos/nns/nns-app-quickstart#vote-on-proposals).
+
+## Liquid democracy / Following
+Users may not have the time or knowledge to participate in all voting decisions. Therefore, instead of directly voting on proposals, neuron holders may choose to delegate their vote to other neurons that they trust with certain decisions. This concept of delegating the right to vote to other voters who then effectively vote with more voting power is called **liquid democracy**. In the NNS DAO, this is also called **following**, as a neuron follows the decision of some other neurons. The liquid democracy allows even neurons that do not have enough knowledge of a certain topic to participate in governance by choosing the neurons that they trust with certain decisions.
```

Comment:
```suggestion
Users may not have the time or knowledge to participate in all voting decisions. Therefore, instead of directly voting on proposals, neuron holders may choose to delegate their vote to other neurons that they trust with certain decisions. This concept of delegating the right to vote to other neurons, who then effectively vote with more voting power, is called **liquid democracy**. In the NNS DAO, this is also called **following**, as a neuron follows the decision of some other neurons.
```